buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1022,6 +1022,50 @@ class BeamModulePlugin implements Plugin<Project> {
         }
       }
 
+      // Ban these dependencies from all configurations
+      project.configurations.all {
```

Comment:
Why did you need/want to move these up to here?

Reply:
`Cannot change dependencies of dependency configuration ':sdks:java:harness:provided' after it has been resolved.` This changes all configurations.